client/tools/webpack.config.common.js

```diff
@@ -74,8 +80,8 @@ function getPlugins(isProduction) {
       filename: path.join(config.buildDir, "index.html"),
       template: config.indexHtmlTemplate,
       // minify: isProduction ? {} : false
-    }),
-    new HtmlWebpackPolyfillIOPlugin({ features: "es6,fetch" }),
+    })
+    // new HtmlWebpackPolyfillIOPlugin({ features: "es6,fetch" }),
```

Comment:
I don't know too much about this - but I suspect we might need this to support some older browsers?